target/linux/ath79/dts/qca9533_tplink_tl-wr841hp-v3.dts

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr841hp-v3", "qca,qca9533";
+	model = "TP-Link TL-WR841HP v3";
+
+	aliases {
+		label-mac-device = &wmac;
+		led-boot = &led_pwr;
+		led-failsafe = &led_pwr;
+		led-running = &led_pwr;
+		led-upgrade = &led_pwr;
+	};
+
+	leds: leds {
```

Comment:
The "leds:" DT label can be dropped here.

Reply:
Did you mean like this ? :
leds: {
	compatible = "gpio-leds";
	wifi {

Reply:
Just

```
leds {
	...
```

Reply:
ok, I understand, tks.